canonical-websites/www.ubuntu.com
# 3356 kubernetes overview page

## Done

Update `/kubernetes`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/kubernetes>
- See that it matches the [copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#)


## Issue / Card

Fixes #3356
